README.md

```diff
@@ -1,76 +1,94 @@
 # Blue Core API
 
-API for managing Blue Core resources and workflows using PostgreSQL and Airflow platforms.
+Blue Core API is a REST API for managing Blue Core resources and workflows using PostgreSQL and Airflow platforms. The application relies on Keycloak and Airflow services. To understand the production deployment of Blue Core API you'll want to look at the [Terraform](https://github.com/blue-core-lod/terraform) repository.
 
+Read on for instructions about how to develop this codebase.
 
-## 🗂️ Project structure
-```
-bluecore_store_migrations/
-|  |── env.py
-|  |── versions/
-|
-src/bluecore_api/
-│   │
-│   ├── app/
-│   │   ├── __init__.py
-│   │   ├── main.py
-│   │   ├── config.py
-│   │   
-│   │  
-│   └── models.py
-tests/
-|
-├── pyproject.yaml
-└── README.md
+## 🧰 Prerequisites
+- [uv]
+- [Docker]
+
+## Setup PostgreSQL, Keycloak and Airflow
+
+Blue Core API depends on having PostgreSQL, Airflow and Keycloak running. The easiest way to do this is to clone the [Blue Core Workflows] repository, and start the default configuration:
+
+```shell
+git clone https://github.com/blue-core-lod/bluecore-workflows
+cd bluecore-workflows
+docker compose up 
 ```
 
----
+## Database Models
 
-## 🛠️ Installation
+The Blue Core API depends on database models in the [Blue Core Data Models] to be present and up to date.
```

Comment:
I think the `[Blue Core Data Models]` is missing the URL `[BLUECORE_MODELS] `

Reply:
I moved the URLs to the bottom of the doc, I think it's rendering ok?

https://github.com/blue-core-lod/bluecore_api/blob/5c0af84c36b4f8b0b3cfe67c46c8e1bcdcf7ec0b/README.md